packages/components/package.json

```diff
@@ -98,43 +98,43 @@
     "@types/vis": "4.21.16",
     "@typescript-eslint/eslint-plugin": "2.19.0",
     "@typescript-eslint/parser": "2.19.0",
-    "babel-loader": "8.0.6",
+    "babel-loader": "8.1.0",
     "bootstrap-sass": "3.4.1",
-    "cross-env": "5.2.0",
+    "cross-env": "7.0.2",
     "css-loader": "3.4.2",
-    "enzyme": "3.10.0",
-    "enzyme-adapter-react-16": "1.14.0",
-    "enzyme-to-json": "3.3.5",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "enzyme-to-json": "3.4.4",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.10.0",
     "eslint-import-resolver-typescript": "2.0.0",
     "eslint-plugin-import": "2.20.1",
     "eslint-plugin-jest": "23.6.0",
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.18.3",
-    "file-loader": "5.1.0",
-    "gh-pages": "2.0.1",
+    "file-loader": "6.0.0",
+    "gh-pages": "2.2.0",
     "jest": "24.9.0",
     "jest-cli": "24.9.0",
     "jest-teamcity-reporter": "0.9.0",
     "jsdom": "15.1.1",
-    "mini-css-extract-plugin": "0.8.0",
-    "node-sass": "4.12.0",
+    "mini-css-extract-plugin": "0.9.0",
+    "node-sass": "4.13.1",
     "nps": "5.9.12",
     "nps-utils": "1.7.0",
     "prettier": "1.19.1",
     "react-bootstrap": "0.32.4",
     "react-test-renderer": "16.8.6",
-    "sass-loader": "7.1.0",
+    "sass-loader": "8.0.2",
     "ts-jest": "24.0.2",
-    "ts-loader": "6.0.2",
-    "typedoc": "0.14.2",
+    "ts-loader": "6.2.2",
+    "typedoc": "0.17.3",
     "typescript": "3.7.3",
```

Comment:
Did you attempt an update of TypeScript and run into errors?

Reply:
Yes, I ran into some errors when updating typescript to 3.8.3:
```
 FAIL  src/components/files/actions.spec.ts
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/components/files/actions.spec.ts:181:11 - error TS2739: Type '{ size: number; type: string; lastModified: number; name: string; slice: (start: number, end: number, compareType: string) => Blob; }' is missing the following properties from type 'File': arrayBuffer, stream, text
```
I thought this was within the application code, but since it's only in the tests, I updated with some undefined values for those fields.  Seems to be ok now (locally).